quic/s2n-quic/src/provider.rs

```diff
@@ -52,6 +52,15 @@ cfg_if!(
     }
 );
 
+cfg_if!(
+    if #[cfg(any(test, feature = "unstable-provider-dc"))] {
```

Comment:
Are you wanting to add this feature in this PR?

Reply:
knew I was forgetting something :-)

Reply:
also needed to add the provider to the server and client builders